src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -188,6 +275,51 @@ struct alignas(4) IMUMeasurement : public MessagePayload {
   double gyro_std_rps[3] = {NAN, NAN, NAN};
 };
 
+/**
+ * @brief Raw (uncorrected) IMU sensor measurement output (@ref
+          MessageType::RAW_IMU_OUTPUT, version 1.0).
```

Comment:
Missmatch between comment version and message version.

Reply:
Oh I see I guess we've been calling version 0, 1.0? I guess we're consistent with that, though not with what to call MESSAGE_VERSION = 1. We usually refer to that as 1.1, but it's wrong about half the time in RelativeENUPositionMessage, CalibrationStatusMessage, and GetConfigMessage .

Reply:
Yeah basically the system is "this struct is 1.x and the version field is the x". If we want to go to 2.x it'll be a new struct. We haven't actually had to do that yet so not sure how it'll look if we do.

If the constant doesn't match up with the documentation, that's a mistake and we should fix it.